sdk/src/main/java/com/karhoo/sdk/analytics/Event.kt

```diff
@@ -43,7 +43,25 @@ enum class Event constructor(val value: String) {
     USER_PROFILE_SAVE_PRESSED("user_profile_save_pressed"),
     USER_PROFILE_UPDATE_SUCCESS("user_profile_update_success"),
     USER_PROFILE_UPDATE_FAILED("user_profile_update_failed"),
+
+    //PAYMENT
+    CARD_AUTHORISATION_SUCCESS("CARD_AUTHORISATION_SUCCESS"),
+    CARD_AUTHORISATION_FAILURE("CARD_AUTHORISATION_FAILURE"),
     PAYMENT_FAILED("payment_failed"),
+
+    //LOYALTY
+    LOYALTY_STATUS_REQUESTED("LOYALTY_STATUS_REQUESTED"),
+    LOYALTY_PREAUTH_FAILED("LOYALTY_PREAUTH_FAILURE"),
+    LOYALTY_PREAUTH_SUCCESS("LOYALTY_PREAUTH_SUCCESS"),
+
+    //CHECKOUT
+    CHECKOUT_SCREEN("checkout_screen"),
+
+    //BOOKING
+    BOOKING_REQUESTED("BOOKING_REQUESTED"),
+    BOOKING_SUCCESS("BOOKING_SUCCESS"),
+    BOOKING_FAILURE("BOOKING_FAILED"),
```

Comment:
again failure + failed. should be failure

Reply:
Corrected